src/Bicep.LangServer/LangServerConstants.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Bicep.LanguageServer
+{
+    public static class LangServerConstants
+    {
+        public const string Build = "build";
```

Comment:
Very vague names, no idea how these are used from this.

Reply:
Updated to BuildCommand and DeployCommand as part of this commit:
https://github.com/Azure/bicep/commit/32bed56af070ae3b2c2b494c956f07cf68874667

Let me know if you have any other suggestions.

Thanks!